tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "target": "es5",
     "lib": [
       "dom",
-      "es7"
+      "es2017"
```

Comment:
Could you please explain, what are the benefits of this change? Thank you!

Reply:
This allows to use features of es2017. In this case I used the `Object.values` method.